src/api/app/models/bs_request_permission_check.rb

```diff
@@ -385,10 +385,10 @@ def set_permissions_for_action(action, new_state = nil)
         @write_permission_in_this_action = true
       end
     else
-      if @target_project && User.session!.can_create_package_in?(@target_project, true)
+      if @target_project && PackagePolicy.new(User.session!, Package.new(project: @target_project), true).create?
```

Comment:
I cannot use `Pundit.policy(...)` here since there is a third argument in the call.